VideoLocker/src/main/java/org/edx/mobile/module/notification/NotificationDelegate.java

```diff
@@ -43,4 +43,11 @@
      * @param subscribe
      */
     public void toggleSubscribeToNotificationServer(final String channel, boolean subscribe);
```

Comment:
I guess I missed this when it went in, but "toggle" usually means it inverts the current value. Maybe this should just be ``toggleSubscribeToNotificationServer`` or perhaps setSubscribeToNotificationChannel()